internal/selectors/selectors.go

```diff
@@ -158,19 +158,43 @@ func GetNoneSelector(inst *unstructured.Unstructured) (bool, error) {
 	return noneSelector, nil
 }
 
-// cmExclusions are known (istio and kube-root) CA configmap which are excluded from propagation
-var cmExclusions = []string{"istio-ca-root-cert", "kube-root-ca.crt"}
+// cmExclusionsByName are known (istio and kube-root) CA configmap which are excluded from propagation
+var cmExclusionsByName = []string{"istio-ca-root-cert", "kube-root-ca.crt"}
+
+// A label as a key, value pair, used to exclude resources matching this label (both key and value).
+type ExclusionByLabelsSpec struct {
+	Key   string
+	Value string
+}
+
+// ExclusionByLabelsSpec are known label key-value pairs which are excluded from propagation. Right
+// now only used to exclude resources created by Rancher, see "System Tools > Remove"
+// (https://rancher.com/docs/rancher/v2.6/en/system-tools/#remove)
+var exclusionByLabels = []ExclusionByLabelsSpec{
+	{Key: "cattle.io/creator", Value: "norman"},
+}
 
 // isExcluded returns true to indicate that this object is excluded from being propagated
 func isExcluded(inst *unstructured.Unstructured) (bool, error) {
 	name := inst.GetName()
 	kind := inst.GetKind()
 	group := inst.GroupVersionKind().Group
+	labels := inst.GetLabels()
 
-	for _, excludedResourceName := range cmExclusions {
+	// exclusion by name
+	for _, excludedResourceName := range cmExclusionsByName {
 		if group == "" && kind == "ConfigMap" && name == excludedResourceName {
 			return true, nil
 		}
 	}
+
+	// exclusion by labels
+	for _, res := range exclusionByLabels {
+		gotLabelValue, ok := labels[res.Key]
```

Comment:
Maybe comment that label values can be empty so we need to check for presence explicitly (I actually thought we could simplify this to `if labels[res.Key] == res.Value`, but I was wrong, and I don't want someone else to "simplify" this in the future by mistake).

Reply:
Yeah, valid point. done!